src/libs/OptionsListUtils.ts

```diff
@@ -1815,9 +1816,9 @@ function getFilteredOptions(
  */
 
 function getShareDestinationOptions(
-    reports: Record<string, Report>,
+    reports: Record<string, Report | null>,
     personalDetails: OnyxEntry<PersonalDetailsList>,
-    betas: Beta[] = [],
+    betas: Beta[] | null = [],
```

Comment:
```suggestion
    betas: OnyxEntry<Beta[]>= [],
```

shouldn't be OnyxEntry?